src/lyrics/lyrics.h

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2010-2019 Ariadne Conill <ariadne@dereferenced.org>
+ * Copyright (c) 2024 Thomas Lange <thomas-lange2@gmx.de>
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
+ * IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef AUDACIOUS_LYRICS_H
+#define AUDACIOUS_LYRICS_H
+
+#include <string.h>
+#include <libxml/parser.h>
+
+#include <libaudcore/audstrings.h>
+#include <libaudcore/i18n.h>
+#include <libaudcore/runtime.h>
+#include <libaudcore/vfs.h>
+#include <libaudcore/vfs_async.h>
+
+struct LyricsState {
+    String filename; // of song file
+    String title, artist;
+    String lyrics;
+
+    enum Source {
+        None,
+        Embedded,
+        Local,
+        LyricWiki, // TODO: Can this be safely removed?
```

Comment:
It is no longer used, but affects the value of entries below. Can we remove it anyway?

Reply:
I think we can remove `LyricWiki`. It doesn't look like `enum Source` is used directly in the config file.